### DISCUSSION TITLE
People explore timeline in my samsung android phone is blocked at 2021 (no issue with web app)

### DISCUSSION THREAD

**Discussion by olivier-da** [immich-app/immich#5787] [Q&A]
Hello,

I have an issue with the people explore timeline in my samsung android phone
In mobile app the timeline of explore page for people stops at 2021 whereas I uploaded with CLI my photos from now to 2014
In the main page of mobile all the photos are well displayed. The issue is only in explore page by people

In Web app, there are no issues I can see all the photos in explore people page

I've uninstalled the mobile app and reinstalled but same issue.

Any help would be appreciated :-)

Big thank you for all the developpers. Immich is really amazing

**Comment by olivier-da** [comment]
Nobody has the same issue? 
Only a subset of most recent photos in explore by person menu is displayed compared to web app in browser?
for me:
android mobile app: explore by person photos: 2023 to 2021
web app in browser PC: 2023 to 2010

Same issue with the android app on my wife's phone AND one of my colleague

Any clue?

**Comment by martabal** [comment]
I think this is a limitation of the mobile. It uses only 1000 assets
